src/libraries/System.Net.Mail/src/System/Net/Base64Stream.cs

```diff
@@ -80,14 +78,14 @@ public override void Close()
             base.Close();
         }
 
-        public unsafe int DecodeBytes(byte[] buffer, int offset, int count)
+        public unsafe int DecodeBytes(Span<byte> buffer)
```

Comment:
Something to consider for the future if we're cleaning this up is to try and replace all of this with built-in base64 helpers.